ui/app.js

```diff
@@ -669,7 +669,7 @@ $(document).ready(() => {
     ".manage-incidents-create",
     function () {
       let tempalte =
-        "📝 Summary:\n\n[Insert Report Summary Here]\n\n🧍 Hostage: [Name Here]\n\n🔪 Weapons/Items Confiscated:\n\n· [Insert List Here]\n\n-----\n💸 Fine:\n⌚ Sentence:\n-----";
+        "Summary:\n\n[Insert Report Summary Here]";
```

Comment:
Please re-add this, since we are merging into the main branch, we want to keep all original things and just leave the current edit